.github/workflows/push.yml

```diff
@@ -31,16 +31,17 @@ env:
 
 jobs:
   build:
-    name: Build Image
+    name: Calculate version
     runs-on: ubuntu-latest
     steps:
     - name: Check out code
       uses: actions/checkout@v2
       with:
           fetch-depth: 0
-    - name: Setup Go
-      uses: actions/setup-go@v2
-      with:
-        go-version: '1.17'
     - name: calculate version
-      uses: ./
\ No newline at end of file
+      id: version
+      uses: ./
+    - name: Push version
+      run: |
+        git tag ${{ steps.version.outputs.tag }}
+        git push --tags
```

Comment:
we should not push tags here
Only push tags at the end of pipeline after testing and build are OK

Reply:
Currently the tests are not implemented, this is just a test pipeline. I can remove the push tags though.